qemu.go

```diff
@@ -106,6 +108,11 @@ const (
 	maxDevIDSize = 31
 )
 
+const (
+	// NVDIMM device needs memory space (1024MB is suggested by @grahamwhaley)
```

Comment:
looks correct to me :-) (yes, github nicely hides the full URL where it can :-) )